.reuse/dep5

```diff
@@ -0,0 +1,17 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Slang Binaries
+Upstream-Contact: The Khronos Group, Inc.
+Source: https://github.com/shader-slang/slang-binaries
+
+Files: *
+Copyright: 2016, Carnegie Mellon University and NVIDIA Corporation
+           2024 The Khronos Group, Inc.
+License: Apache-2.0 WITH LLVM-exception
```

Comment:
we should limit these files to just the ones we have authored, this definitely doesn't apply to the binaries in here.